api_test.go

```diff
@@ -29,20 +26,25 @@ import (
 	"testing"
 	"time"
 
-	"github.com/gofrs/flock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	yaml "gopkg.in/yaml.v2"
 )
 
+// tests against cacophony-api require apiURL to be pointing
+// to a valid cacophony-api server and test-seed.sql to be run
+
 var apiURL = "http://localhost:1080"
 var tokenSuccess = true
 var responseHeader = http.StatusOK
 var rawThermalData = randString(100)
+var testConfig = "/var/tmp/go-api-test-config.yaml"
```

Comment:
instead of assuming /var/tmp is the right place,  a better thing to do is use `TempDir` or `TempFile` in `io/ioutil`. Get `createTestConfig` to use one of these and return the name of the file.